spec/run_spec.js

```diff
@@ -0,0 +1,54 @@
+describe('run', function() {
+  var path = require('path'),
+      runJasmine = require('../lib/run');
+
+  it('should go through each jasmine method', function() {
```

Comment:
From this description, I can't tell _why_ we're going "through each jasmine method". Please expand.